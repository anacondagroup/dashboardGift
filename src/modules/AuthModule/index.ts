export { default } from './components/AuthModule';
