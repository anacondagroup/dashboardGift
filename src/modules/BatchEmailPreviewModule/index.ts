export { default } from './components/BatchEmailPreviewModule';
