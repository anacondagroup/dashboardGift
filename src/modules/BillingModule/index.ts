export { default } from './components/BillingModule';
