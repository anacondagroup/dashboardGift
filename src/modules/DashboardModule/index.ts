export { default } from './components/DashboardModule';
