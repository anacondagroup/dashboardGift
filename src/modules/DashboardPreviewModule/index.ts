export { default } from './components/DashboardPreviewModule';
