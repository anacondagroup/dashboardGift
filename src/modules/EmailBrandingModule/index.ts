export { default } from './components/EmailBrandingModule';
