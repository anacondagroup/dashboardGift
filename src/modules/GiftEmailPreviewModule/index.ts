export { default } from './components/GiftEmailPreviewModule';
