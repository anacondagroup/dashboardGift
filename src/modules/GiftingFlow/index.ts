export { default } from './components/GiftingFlow';
