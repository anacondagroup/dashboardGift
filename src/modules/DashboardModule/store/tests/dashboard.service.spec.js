import { stateToUrl, urlToState } from '@alycecom/modules';

describe('Dashboard service', () => {
  describe('urlToState', () => {
    it('Should rename keys', () => {
      const actual = urlToState({
        date_range_from: '',
        date_range_to: '',
        teams_search: '',
        teams_sort: '',
        teams_direction: '',
        team_members_page: '',
        team_members_search: '',
        team_members_sort: '',
        team_members_direction: '',
        gift_sort: '',
        gift_direction: '',
        gift_search: '',
        gift_page: '',
        campaigns_search: '',
        campaigns_sort: '',
        campaigns_direction: '',
        contacts_search: '',
        contacts_sort: '',
        contacts_direction: '',
        contacts_page: '',
      });

      expect(actual).toMatchSnapshot();
    });

    describe('stateToUrl', () => {
      it('Should rename keys', () => {
        const actual = stateToUrl({
          campaignsSearch: '',
          campaignsSort: '',
          campaignsDirection: '',
          dateRangeFrom: '',
          dateRangeTo: '',
          teamsSearch: '',
          teamsSort: '',
          teamsDirection: '',
          teamMembersPage: '',
          teamMembersSearch: '',
          teamMembersSort: '',
          teamMembersDirection: '',
          giftSort: '',
          giftDirection: '',
          giftSearch: '',
          giftPage: '',
          contactsSearch: '',
          contactsSort: '',
          contactsDirection: '',
          contactsPage: '',
        });

        expect(actual).toMatchSnapshot();
      });
    });
  });
});
