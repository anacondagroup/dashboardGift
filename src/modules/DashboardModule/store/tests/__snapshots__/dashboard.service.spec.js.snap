// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard service urlToState Should rename keys 1`] = `
Object {
  "campaignsDirection": "",
  "campaignsSearch": "",
  "campaignsSort": "",
  "contactsDirection": "",
  "contactsPage": "",
  "contactsSearch": "",
  "contactsSort": "",
  "dateRangeFrom": "",
  "dateRangeTo": "",
  "giftDirection": "",
  "giftPage": "",
  "giftSearch": "",
  "giftSort": "",
  "teamMembersDirection": "",
  "teamMembersPage": "",
  "teamMembersSearch": "",
  "teamMembersSort": "",
  "teamsDirection": "",
  "teamsSearch": "",
  "teamsSort": "",
}
`;

exports[`Dashboard service urlToState stateToUrl Should rename keys 1`] = `
Object {
  "campaigns_direction": "",
  "campaigns_search": "",
  "campaigns_sort": "",
  "contacts_direction": "",
  "contacts_page": "",
  "contacts_search": "",
  "contacts_sort": "",
  "date_range_from": "",
  "date_range_to": "",
  "gift_direction": "",
  "gift_page": "",
  "gift_search": "",
  "gift_sort": "",
  "team_members_direction": "",
  "team_members_page": "",
  "team_members_search": "",
  "team_members_sort": "",
  "teams_direction": "",
  "teams_search": "",
  "teams_sort": "",
}
`;
